charts/csm-authorization/Chart.yaml

```diff
@@ -8,11 +8,11 @@ description: |
   storage enablers for Dell EMC storage products.  CSM for Authorization provides storage and Kubernetes administrators the ability
   to apply RBAC for Dell CSI Drivers.
 dependencies:
-  - name: cert-manager
-    version: 1.11.0
-    repository: https://charts.jetstack.io
-    condition: cert-manager.enabled
-  - name: ingress-nginx
-    version: 4.0.19
-    repository: https://kubernetes.github.io/ingress-nginx
-    condition: ingress-nginx.enabled
+- name: cert-manager
```

Comment:
Can you fix linting on this file?

Reply:
Reverted cert-manager to 1.10.0 like we have been using.